supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/GpuValidation.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.backend.impl.jes
+
+import cats.syntax.validated._
+import com.typesafe.config.Config
+import common.validation.ErrorOr.ErrorOr
+import cromwell.backend.validation.{IntRuntimeAttributesValidation, OptionalRuntimeAttributesValidation, RuntimeAttributesValidation}
+import wom.RuntimeAttributesKeys.{GpuKey, GpuMaxKey, GpuMinKey}
+import wom.types.WomIntegerType
+import wom.values.{WomInteger, WomValue}
+
+object GpuValidation {
+  lazy val instance: RuntimeAttributesValidation[Int] = new GpuValidation(GpuKey)
+  lazy val optional: OptionalRuntimeAttributesValidation[Int] = instance.optional
+  lazy val instanceMin: RuntimeAttributesValidation[Int] = new GpuValidation(GpuMinKey)
+  lazy val optionalMin: OptionalRuntimeAttributesValidation[Int] = instanceMin.optional
+  lazy val instanceMax: RuntimeAttributesValidation[Int] = new GpuValidation(GpuMaxKey)
+  lazy val optionalMax: OptionalRuntimeAttributesValidation[Int] = instanceMax.optional
+
+  lazy val defaultMin: WomValue = WomInteger(0)
+  def configDefaultWomValue(config: Option[Config]): Option[WomValue] = instance.configDefaultWomValue(config)
+}
+
+class GpuValidation(attributeName: String) extends IntRuntimeAttributesValidation(attributeName) {
+  override protected def validateValue: PartialFunction[WomValue, ErrorOr[Int]] = {
+    case womValue if WomIntegerType.coerceRawValue(womValue).isSuccess =>
+      WomIntegerType.coerceRawValue(womValue).get match {
+        case WomInteger(value) =>
+          if (value.toInt < 0)
```

Comment:
ToL Someday I'd like to use `Int Refined Positive` and [this lib](https://github.com/carlpulley/validated-config) to do these validations for us

Reply:
The runtime attribute validation code pre-dates our use of Refined and I mostly copied this from somewhere else 🙈

Reply:
Yep I know!  Just wanted to plant a seed.